index.html

```diff
@@ -4,15 +4,86 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    
     <!-- Add a link to your custom CSS here -->
 
+    <link rel="stylesheet" href="css/main.css">
     <title>Responsive grid project</title>
 </head>
 
 <body>
 
     <!-- Add your markup here -->
+<main class="h-main">
 
+     <section class="h-hero">
```

Comment:
Is this a hero? What does hero mean in the context of web design?

https://uxplanet.org/unsung-heros-of-ui-look-at-hero-images-in-web-design-881e04fe7bd2

Reply:
Thanks for emphasising 'hero' concept. 
As per original design, this section should be 'hero' .